source/synthDrivers/oneCore.py

```diff
@@ -81,6 +84,13 @@ def convertCharacterModeCommand(self, command):
 		# Therefore, we don't use it.
 		return None
 
+	def convertLangChangeCommand(self, command):
+		lcid = languageHandler.localeNameToWindowsLCID(command.lang)
+		if lcid in (0, languageHandler.LOCALE_CUSTOM_UNSPECIFIED):

```

Comment:
In that case I think its best to keep checking for it and logging. It might be worth adding the return value to the log message. I guess we could name the zero something like `INVALID_PARAM_VALUES` or include some of what you said here as a comment to save the next person from having to look up what zero means.